docs/source/extensions_tutorial/extension_examples/labmetadata_extension.rst

```diff
@@ -215,7 +215,7 @@ Creating unit tests
                 cd ndx-labmetadata-example
                 pytest
 
-    .. tab:: MatLab
+    .. tab:: MATLAB
```

Comment:
@rly can you also change the other spellings of Matlab on the page for consistency